include/sway/tree/workspace.h

```diff
@@ -110,14 +110,14 @@ void workspace_unwrap_children(struct sway_workspace *ws,
 
 void workspace_detach(struct sway_workspace *workspace);
 
-void workspace_add_tiling(struct sway_workspace *workspace,
+struct sway_container *workspace_add_tiling(struct sway_workspace *workspace,
 		struct sway_container *con);
 
 void workspace_add_floating(struct sway_workspace *workspace,
 		struct sway_container *con);
 
-void workspace_insert_tiling(struct sway_workspace *workspace,
-		struct sway_container *con, int index);
+struct sway_container *workspace_insert_tiling(struct sway_workspace *workspace,
+		struct sway_container *con, int index, enum sway_container_layout layout);
```

Comment:
I added this layout argument because I needed a fucntion that performed the insert but without the potential split for workspace_layout.

Since this ended up only being used in one place with a layout other than config->default_layout I'd rather remove it and make that a different function.

Reply:
Yeah, that sounds like a good idea.

Reply:
Yeah, that sounds like a good idea.